src/main/java/org/mockito/internal/matchers/Equals.java

```diff
@@ -5,16 +5,16 @@
 
```

Comment:
@ChristianSchwarz 
If you post an issue related to implement a `Equals` and other non-generic mathcers, please let me know, I would like to contribute. It's really bothering me that I have a lot of warnings.
Thanks